packages/auth/env.ts

```diff
@@ -59,32 +58,5 @@ export const env = createEnv({
         }
       : {}),
   },
-  client: {},
-  runtimeEnv: {
-    AUTH_LOGOUT_REDIRECT_URL: process.env.AUTH_LOGOUT_REDIRECT_URL,
-    AUTH_SESSION_EXPIRY_TIME: process.env.AUTH_SESSION_EXPIRY_TIME,
-    AUTH_PROVIDERS: process.env.AUTH_PROVIDERS,
-    AUTH_LDAP_BASE: process.env.AUTH_LDAP_BASE,
-    AUTH_LDAP_BIND_DN: process.env.AUTH_LDAP_BIND_DN,
-    AUTH_LDAP_BIND_PASSWORD: process.env.AUTH_LDAP_BIND_PASSWORD,
-    AUTH_LDAP_GROUP_CLASS: process.env.AUTH_LDAP_GROUP_CLASS,
-    AUTH_LDAP_GROUP_FILTER_EXTRA_ARG: process.env.AUTH_LDAP_GROUP_FILTER_EXTRA_ARG,
-    AUTH_LDAP_GROUP_MEMBER_ATTRIBUTE: process.env.AUTH_LDAP_GROUP_MEMBER_ATTRIBUTE,
-    AUTH_LDAP_GROUP_MEMBER_USER_ATTRIBUTE: process.env.AUTH_LDAP_GROUP_MEMBER_USER_ATTRIBUTE,
-    AUTH_LDAP_SEARCH_SCOPE: process.env.AUTH_LDAP_SEARCH_SCOPE,
-    AUTH_LDAP_URI: process.env.AUTH_LDAP_URI,
-    AUTH_OIDC_CLIENT_ID: process.env.AUTH_OIDC_CLIENT_ID,
-    AUTH_OIDC_CLIENT_NAME: process.env.AUTH_OIDC_CLIENT_NAME,
-    AUTH_OIDC_CLIENT_SECRET: process.env.AUTH_OIDC_CLIENT_SECRET,
-    AUTH_OIDC_ISSUER: process.env.AUTH_OIDC_ISSUER,
-    AUTH_OIDC_SCOPE_OVERWRITE: process.env.AUTH_OIDC_SCOPE_OVERWRITE,
-    AUTH_OIDC_GROUPS_ATTRIBUTE: process.env.AUTH_OIDC_GROUPS_ATTRIBUTE,
-    AUTH_LDAP_USERNAME_ATTRIBUTE: process.env.AUTH_LDAP_USERNAME_ATTRIBUTE,
-    AUTH_LDAP_USER_MAIL_ATTRIBUTE: process.env.AUTH_LDAP_USER_MAIL_ATTRIBUTE,
-    AUTH_LDAP_USERNAME_FILTER_EXTRA_ARG: process.env.AUTH_LDAP_USERNAME_FILTER_EXTRA_ARG,
-    AUTH_OIDC_AUTO_LOGIN: process.env.AUTH_OIDC_AUTO_LOGIN,
-    AUTH_OIDC_NAME_ATTRIBUTE_OVERWRITE: process.env.AUTH_OIDC_NAME_ATTRIBUTE_OVERWRITE,
-  },
-  skipValidation,
-  emptyStringAsUndefined: true,
+  experimental__runtimeEnv: process.env,
```

Comment:
Are we sure that we want to use this experimental feature?

Reply:
IMO it is fine to use this experimental feature as it is described as the following:

```
Can be used for Next.js ^13.4.4 since they stopped static analysis of server side process.env. Only client side process.env is statically analyzed and needs to be manually destructured.
```
As we are not able to use client side env variables, it is okay to use it